support-lib/jni/djinni_support.hpp

```diff
@@ -567,13 +567,45 @@ class JniEnum {
 
 protected:
     JniEnum(const std::string & name);
+    jclass enumClass() const { return m_clazz.get(); }
 
 private:
     const GlobalRef<jclass> m_clazz;
     const jmethodID m_staticmethValues;
     const jmethodID m_methOrdinal;
 };
 
+class JniFlags : public JniEnum {
```

Comment:
Does this need to inherit from JniEnum, rather than just having one as a member, or using it internally?
I feel like the inherited functions like the overload of create(), and the invalid ordinal() method are confusing.